# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/TransportSecurityInfoUtils.cpp
# Commit: b45bd4ea0000
# Full Hash: b45bd4ea0000e64e4b159cac01d09519d73166f7
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-11-17 09:39:01
# Regressor Bug: 1791633
# File Overlap Count: 1
# Description:
#   Bug 1791633 - separate nsITLSSocketControl from nsITransportSecurityInfo r=necko-reviewers,kershaw,jschanck
#   
#   Depends on D160311
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160313
# ==============================================================================

diff -r 705439f1d269 -r b45bd4ea0000 ipc/glue/TransportSecurityInfoUtils.cpp
--- a/ipc/glue/TransportSecurityInfoUtils.cpp	Wed Nov 16 16:37:29 2022 +0000
+++ b/ipc/glue/TransportSecurityInfoUtils.cpp	Wed Nov 16 16:37:29 2022 +0000
@@ -6,6 +6,7 @@
 
 #include "ipc/IPCMessageUtils.h"
 #include "mozilla/psm/TransportSecurityInfo.h"
+#include "nsNSSCertificate.h"
 
 namespace IPC {
 